command/plan.go

```diff
@@ -129,12 +112,18 @@ func (c *PlanCommand) Run(args []string) int {
 	}
 
 	// Force the region to be that of the job.
-	if r := job.Region; r != "" {
+	if r := *job.Region; r != "" {
```

Comment:
This can be a nil pointer dereference since the API job has not been canonicalized